src/main/java/com/amazon/opendistroforelasticsearch/security/auth/UserInjector.java

```diff
@@ -66,13 +67,23 @@
 
     }
 
-    public static class InjectedUser extends User {
+    static class InjectedUser extends User {
```

Comment:
@vrozov I understand this change has been merged. But out of curiosity, why was this class defined as a public? You removed "public" because it doesn't need to be public or being public could cause any side effect?